FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -376,10 +361,14 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <additions>
                             <list>
                                 <item xml:id="a1">
+                                    <desc type="OwnershipNote">The name of the owner <ref target="PRS13990Semon">ስምዖን</ref> has been inscribed various times in most of the manuscript parts, 
```

Comment:
maybe better as an extra (e1) and you can surely copy paste the loci here not to lose the already encoded information

Reply:
Yes, as I wrote, the current convention is to include this information in extras. This specifically does not mean that it is a later addition!

Reply:
Done with all the loci in one extra (e1)